ddtrace/appsec/_iast/_taint_tracking/initializer/initializer.cpp

```diff
@@ -5,11 +5,6 @@
 using namespace std;
```

Comment:
If `initializer.cpp` doesn't have a `initializer.h` anymore, then typically this should be a `.hpp` file.